### PR TITLE
owners: add puerco/cpanato to the sig-release-leads

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:

As part of the onboard described in this issue: https://github.com/kubernetes/sig-release/issues/1590

/assign @justaugustus @saschagrunert @jeremyrickard @hasheddan

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -228,18 +229,18 @@ aliases:
   #   k/test-infra jobs and overall CI Signal
   #
   release-engineering-approvers:
-    - alejandrox1 # SIG Technical Lead / emergency approval
-    - cpanato # Release Manager / EU backup approval coverage
-    - hasheddan # Release Manager / US backup approval coverage
+    - cpanato # SIG Technical Lead / Release Manager / EU backup approval coverage
```
Comment (justaugustus) [inline OWNERS_ALIASES:232]:
```suggestion
    - cpanato # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -228,18 +229,18 @@ aliases:
...
+    - hasheddan # SIG Technical Lead / Release Manager / US backup approval coverage
```
Comment (justaugustus) [inline OWNERS_ALIASES:233]:
```suggestion
    - hasheddan # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - justaugustus # SIG Chair / RelEng subproject owner / Release Manager
+    - puerco # SIG Technical Lead / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:235]:
```suggestion
    - puerco # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 4]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - saschagrunert # SIG Technical Lead / RelEng subproject owner / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:236]:
```suggestion
    - saschagrunert # SIG Chair / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 5]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - tpepper # SIG Chair / RelEng subproject owner / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:237]:
```suggestion
```

File: OWNERS_ALIASES [hunk 6]
```diff
@@ -228,18 +229,18 @@ aliases:
...
   release-engineering-reviewers:
-    - cpanato # Release Manager
+    - cpanato # SIG Technical Lead / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:239]:
```suggestion
    - cpanato # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 7]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - feiskyer # Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:240]:
```suggestion
```

File: OWNERS_ALIASES [hunk 8]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - hasheddan # Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:241]:
```suggestion
    - hasheddan # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 9]
```diff
@@ -228,18 +229,18 @@ aliases:
...
-    - hoegaarden # Release Manager
     - justaugustus # SIG Chair / RelEng subproject owner / Release Manager
+    - puerco # SIG Technical Lead / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:243]:
```suggestion
    - puerco # SIG Technical Lead / RelEng subproject owner / Release Manager
```

File: OWNERS_ALIASES [hunk 10]
```diff
@@ -228,18 +229,18 @@ aliases:
...
     - saschagrunert # SIG Technical Lead / RelEng subproject owner / Release Manager
     - tpepper # SIG Chair / RelEng subproject owner / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:245]:
```suggestion
```

File: OWNERS_ALIASES [hunk 11]
```diff
@@ -228,18 +229,18 @@ aliases:
...
   release-engineering-reviewers:
-    - cpanato # Release Manager
+    - cpanato # SIG Technical Lead / Release Manager
     - feiskyer # Release Manager
     - hasheddan # Release Manager
-    - hoegaarden # Release Manager
     - justaugustus # SIG Chair / RelEng subproject owner / Release Manager
+    - puerco # SIG Technical Lead / Release Manager
     - saschagrunert # SIG Technical Lead / RelEng subproject owner / Release Manager
```
Comment (justaugustus) [inline OWNERS_ALIASES:244]:
```suggestion
    - saschagrunert # SIG Chair / RelEng subproject owner / Release Manager
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@cpanato -- Added some code suggestions to clean up further! :)

**Comment by justaugustus** [comment]
/lgtm
/approve
/assign @BenTheElder @spiffxp @fejta 
cc: @kubernetes/sig-release-leads

**Review by BenTheElder — APPROVED** [review]
lgtm

**Pull request merged**